src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ObjectToIdCache.cs

```diff
@@ -15,60 +15,64 @@ internal class ObjectToIdCache
         internal int m_currentCount;
         internal int[] m_ids;
         internal Object[] m_objs;
-
+        internal bool[] m_isWrapped;
+ 
```

Comment:
Nit: There's an extra space here. There are a few more of these throughout this.